src/themes.cpp

```diff
@@ -3186,7 +3191,7 @@ void OS::use_native_fonts() {
 	use_any_font(FL_COURIER, monospace_fonts, _countof(monospace_fonts));
 	// Use common bold monospace font
 	const char *bold_monospace_fonts[] = {
-		"Ubuntu Mono bold", "Noto Sans Mono bold", "Droid Sans Mono bold", "DejaVu Sans Mono bold"
+		"Ubuntu Mono Bold", "Noto Sans Mono Bold", "Droid Sans Mono Bold", "DejaVu Sans Mono Bold"

```

Comment:
This change became necessary after enabling Cairo+Pango drawing on X11 (instead of plain X11 drawing).
This comes from `-D FLTK_GRAPHICS_CAIRO=1` in the `cmake` command.

Enabling Cairo+Pango on X11 isn't strictly necessary, but it avoids some bugs/quirks with plain X11 drawing and it looks better overall.